ada_2_fhir-r4/zibs2020/ada_instance/nl-core-TreatmentObjective-01.xml

```diff
@@ -0,0 +1,37 @@
+<behandeldoel_registratie app="zib2020-nl" shortName="behandeldoel_registratie" formName="behandeldoel" transactionRef="2.16.840.1.113883.2.4.3.11.60.121.4.2.13.5" transactionEffectiveDate="2020-09-01T00:00:00" versionDate="" prefix="zib2017-" language="nl-NL" title="behandeldoel-GewensteGezondheidstoestand" desc="" id="7e7b3f0a-f369-4bc0-bb3f-0af3691bc09b">
+    <behandeldoel conceptId="2.16.840.1.113883.2.4.3.11.60.40.1.13.5.1" id="bd-1">
+        <onderwerp conceptId="2.16.840.1.113883.2.4.3.11.999.121.1.1">
+            <patient-id value="111222333" root="2.16.840.1.113883.2.4.6.3" conceptId="2.16.840.1.113883.2.4.3.11.999.121.1.2"/>
+        </onderwerp>
+        <referenties conceptId="2.16.840.1.113883.2.4.3.11.999.121.1.3">
+            <functionele_of_mentale_status conceptId="2.16.840.1.113883.2.4.3.11.60.40.1.4.26.1" id="fom-1">
+                <referenties conceptId="2.16.840.1.113883.2.4.3.11.999.121.1.3">

```

Comment:
I don't know if layers of referenties will work, but we'll see whenever resolving the MedicalDevice ;)

Reply:
Fixed this with the help of Jorn :)